train/tr10-13B-ml/tr10-13B.slurm

```diff
@@ -46,7 +46,7 @@ GLOBAL_BATCH_SIZE=2048
 
 NLAYERS=40
 NHIDDEN=5120
-NHEADS=32
```

Comment:
I don't why we chose 32? We seem to have updated the `NHIDDEN` value to be 5120 because it was divisible by 128, and `5120 // 128 = 40`.

https://huggingface.slack.com/archives/C01NHER1JLS/p1627034738272600?thread_ts=1626827659.189400&cid=C01NHER1JLS

cc @VictorSanh @stas00 @mryab (People who were involved in the original post)

Reply:
FWIW, 530B training used:
```
NLAYERS=105
NHIDDEN=20480
NHEADS=128
```

So the same proportion as 32 and 5120

Reply:
Also, @TevenLeScao shared elsewhere a research paper showing that many heads we found to quite redundant anyway.

I'm not sure if there is a research showing size of the head vs. number of the heads performance.